app/components/instant-article-field/instant-article-config.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+angular.module('cms.instant_article.config', []).factory('InstantArticleConfig', [function () {
+  var supportedFeatureTypes = [];
+
+  var factory = {};
+
+  factory.setSupportedFeatureTypes = function(featureTypes) {
+    supportedFeatureTypes = featureTypes;
+
+    return this;
+  };
+
+  factory.getSupportedFeatureTypes = function() {
+    return supportedFeatureTypes;
+  };
+
+  return factory;
+}]);
```

Comment:
I think it's a good pattern to have to separate setters and getters, since using the `$get` pattern seals your configuration once all `config` have run.

Also, to keep things consistent, we want to be configuring everything in `config` steps, not `run` steps, unless there's a specific reason we cannot configure during the `config` step.